tests/test_lib.bash

```diff
@@ -4,13 +4,12 @@ tmpdir=$(mktemp -d)
 trap "rm -rf $tmpdir" EXIT
 
 OCRD_CIS_FILEGRP="OCR-D-GT-SEG-LINE"
-# fixme: it does not work like this - the OCR-D GT repo uses different URL paths for different datasets
-# this is merely the path for blumenbach_anatomie_1805.ocrd.zip
-data_url="https://ocr-d-repo.scc.kit.edu/api/v1/dataresources/75ad9f94-dbaa-43e0-ab06-2ce24c497c61/data/"
+data_url="http://hdl.handle.net/21.11156/6B119B3C-A24A-424C-AC3C-27E64B051780"
```

Comment:
please keep the comment – the other scripts attempt to freely combine the base path with the dataset id, which only works as long as the latter stays `blumenbach_anatomie_1805.ocrd.zip`